{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"pytest_version": "7.4.4", "$report_type": "SessionStart"}
{"nodeid": "tests57_49_1.py::test_matrix_of_single_number_and_zeros", "location": ["tests57_49_1.py", 51, "test_matrix_of_single_number_and_zeros"], "keywords": {"test_matrix_of_single_number_and_zeros": 1, "__wrapped__": 1, "tests57_49_1.py": 1, "Folder_49": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00016941595822572708, "start": 1731150837.1609895, "stop": 1731150837.1611595, "$report_type": "TestReport", "item_index": 4, "worker_id": "gw2", "testrun_uid": "6743f42a70834c75a5bb534c19a69076", "node": "<WorkerController gw2>"}
{"nodeid": "tests57_49_1.py::test_matrix_of_several_numbers", "location": ["tests57_49_1.py", 29, "test_matrix_of_several_numbers"], "keywords": {"test_matrix_of_several_numbers": 1, "__wrapped__": 1, "tests57_49_1.py": 1, "Folder_49": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0001423258800059557, "start": 1731150837.1643388, "stop": 1731150837.1644819, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "6743f42a70834c75a5bb534c19a69076", "node": "<WorkerController gw1>"}
{"nodeid": "tests57_49_1.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_49_1.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_49_1.py": 1, "Folder_49": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00015214900486171246, "start": 1731150837.1622837, "stop": 1731150837.1624367, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "6743f42a70834c75a5bb534c19a69076", "node": "<WorkerController gw0>"}
{"nodeid": "tests57_49_1.py::test_matrix_of_single_number_and_zeros", "location": ["tests57_49_1.py", 51, "test_matrix_of_single_number_and_zeros"], "keywords": {"test_matrix_of_single_number_and_zeros": 1, "__wrapped__": 1, "tests57_49_1.py": 1, "Folder_49": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_and_zeros():", "        if -42 == 0:", "            element = 1", "        else:", "            element = 0", "        for i in range(1, 11):", "            mat = ''", "            for j in range(i):", "                row = (f'{element} ' * i) + ';'", "                mat += row", "            mat = mat[:-1]", "            mat = np.matrix(mat)", "            mat[0, 0] = -42", "            if -42 == 0:", "                assert len(submatrix_with_particular_sum(mat)) == 1", "            else:", ">               assert len(submatrix_with_particular_sum(mat)) == i * i"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/tests57_49_1.py", "lineno": 69, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.where(np.sum(matrix, axis=2) == -42, matrix, np.nan).tolist()"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/generated_answer.py", "lineno": 4, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[-42]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_and_zeros():", "        if -42 == 0:", "            element = 1", "        else:", "            element = 0", "        for i in range(1, 11):", "            mat = ''", "            for j in range(i):", "                row = (f'{element} ' * i) + ';'", "                mat += row", "            mat = mat[:-1]", "            mat = np.matrix(mat)", "            mat[0, 0] = -42", "            if -42 == 0:", "                assert len(submatrix_with_particular_sum(mat)) == 1", "            else:", ">               assert len(submatrix_with_particular_sum(mat)) == i * i"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/tests57_49_1.py", "lineno": 69, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.where(np.sum(matrix, axis=2) == -42, matrix, np.nan).tolist()"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/generated_answer.py", "lineno": 4, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[-42]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00029373401775956154, "start": 1731150837.1619487, "stop": 1731150837.1622431, "extras": [], "$report_type": "TestReport", "item_index": 4, "worker_id": "gw2", "testrun_uid": "6743f42a70834c75a5bb534c19a69076", "node": "<WorkerController gw2>"}
{"nodeid": "tests57_49_1.py::test_matrix_of_single_number_and_zeros", "location": ["tests57_49_1.py", 51, "test_matrix_of_single_number_and_zeros"], "keywords": {"test_matrix_of_single_number_and_zeros": 1, "__wrapped__": 1, "tests57_49_1.py": 1, "Folder_49": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.001341876806691289, "start": 1731150837.2316358, "stop": 1731150837.2329786, "$report_type": "TestReport", "item_index": 4, "worker_id": "gw2", "testrun_uid": "6743f42a70834c75a5bb534c19a69076", "node": "<WorkerController gw2>"}
{"nodeid": "tests57_49_1.py::test_matrix_of_several_numbers", "location": ["tests57_49_1.py", 29, "test_matrix_of_several_numbers"], "keywords": {"test_matrix_of_several_numbers": 1, "__wrapped__": 1, "tests57_49_1.py": 1, "Folder_49": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_several_numbers():", "        if -42 <= 0:", "            row = '1 2 3 4 5 6 7 8 9'", "            input_matrix = np.matrix(f'{row}; {row}; {row}')", "        else:", "            row = '-1 -2 -3 -4 -5 -6 -7 -8 -9'", "            input_matrix = np.matrix(f'{row}; {row}; {row}')", ">       assert not submatrix_with_particular_sum(input_matrix)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/tests57_49_1.py", "lineno": 38, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.where(np.sum(matrix, axis=2) == -42, matrix, np.nan).tolist()"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/generated_answer.py", "lineno": 4, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[1, 2, 3, 4, 5, 6, 7, 8, 9],\n        [1, 2, 3, 4, 5, 6, 7, 8, 9],\n        [1, 2, 3, 4, 5, 6, 7, 8, 9]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_several_numbers():", "        if -42 <= 0:", "            row = '1 2 3 4 5 6 7 8 9'", "            input_matrix = np.matrix(f'{row}; {row}; {row}')", "        else:", "            row = '-1 -2 -3 -4 -5 -6 -7 -8 -9'", "            input_matrix = np.matrix(f'{row}; {row}; {row}')", ">       assert not submatrix_with_particular_sum(input_matrix)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/tests57_49_1.py", "lineno": 38, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.where(np.sum(matrix, axis=2) == -42, matrix, np.nan).tolist()"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/generated_answer.py", "lineno": 4, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[1, 2, 3, 4, 5, 6, 7, 8, 9],\n        [1, 2, 3, 4, 5, 6, 7, 8, 9],\n        [1, 2, 3, 4, 5, 6, 7, 8, 9]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00045256316661834717, "start": 1731150837.1647134, "stop": 1731150837.1651669, "extras": [], "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "6743f42a70834c75a5bb534c19a69076", "node": "<WorkerController gw1>"}
{"nodeid": "tests57_49_1.py::test_matrix_of_several_numbers", "location": ["tests57_49_1.py", 29, "test_matrix_of_several_numbers"], "keywords": {"test_matrix_of_several_numbers": 1, "__wrapped__": 1, "tests57_49_1.py": 1, "Folder_49": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00014899298548698425, "start": 1731150837.2398746, "stop": 1731150837.2400246, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "6743f42a70834c75a5bb534c19a69076", "node": "<WorkerController gw1>"}
{"nodeid": "tests57_49_1.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_49_1.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_49_1.py": 1, "Folder_49": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[-42]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[-42]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/tests57_49_1.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.where(np.sum(matrix, axis=2) == -42, matrix, np.nan).tolist()"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/generated_answer.py", "lineno": 4, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[-42]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[-42]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[-42]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/tests57_49_1.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    return np.where(np.sum(matrix, axis=2) == -42, matrix, np.nan).tolist()"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/command_results_1/Folder_49/generated_answer.py", "lineno": 4, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return _wrapreduction(a, np.add, 'sum', axis, dtype, out, keepdims=keepdims,"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 2313, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return reduction(axis=axis, out=out, **passkwargs)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/fromnumeric.py", "lineno": 86, "message": "in _wrapreduction"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return N.ndarray.sum(self, axis, dtype, out, keepdims=True)._collapse(axis)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 320, "message": "in sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def _sum(a, axis=None, dtype=None, out=None, keepdims=False,", "             initial=_NoValue, where=True):", ">       return umr_sum(a, axis, dtype, out, keepdims, initial, where)", "E       numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"], "reprfuncargs": {"args": [["a", "matrix([[-42]])"], ["axis", "2"], ["dtype", "None"], ["out", "None"], ["keepdims", "True"], ["initial", "<no value>"], ["where", "True"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "AxisError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/core/_methods.py", "lineno": 49, "message": "numpy.exceptions.AxisError: axis 2 is out of bounds for array of dimension 2"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.0002245230134576559, "start": 1731150837.162661, "stop": 1731150837.1628864, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "6743f42a70834c75a5bb534c19a69076", "node": "<WorkerController gw0>"}
{"nodeid": "tests57_49_1.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_49_1.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_49_1.py": 1, "Folder_49": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00012284796684980392, "start": 1731150837.2450328, "stop": 1731150837.2451563, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "6743f42a70834c75a5bb534c19a69076", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
